airflow/www/extensions/init_security.py

```diff
@@ -56,14 +56,14 @@ def init_api_experimental_auth(app):
         pass
 
     app.api_auth = []
-    for backend in auth_backends.split(","):
-        try:
+    try:
+        for backend in auth_backends.split(","):
             auth = import_module(backend.strip())
             auth.init_app(app)
             app.api_auth.append(auth)
-        except ImportError as err:
-            log.critical("Cannot import %s for API authentication due to: %s", backend, err)
-            raise AirflowException(err)
+    except ImportError as err:
+        log.critical("Cannot import %s for API authentication due to: %s", backend, err)
+        raise AirflowException(err)
```

Comment:
While we are at this, I wonder if we should remove this exception wrapping. It doesn’t have any value from what I can tell. Same for the one in `__init__.py`.

Reply:
I had the same question because it seems useless. However I prefer to keep this PR changing the syntax without any change in the functionality, I will open some new PR to improve the raised exceptions.